client/src/services/documentService.ts

```diff
@@ -7,7 +7,8 @@ interface DocumentElement {
 
 async function getSampleDocument(): Promise<Array<DocumentElement>> {
     async function getDocumentMd() : Promise<string> {
-      let response = await axios.get("/sample.md")
+      let response = await axios.get("/cleanEHR.md")
```

Comment:
We should figure out how to turn this into a parameter of some kind. Could we perhaps pass the file name in the query string, i.e. you'd navigate to `http://localhost:8080/?cleanEHR` and we'd parse the query in the URL and load the right file?

Reply:
Done in https://github.com/wrattler/wrattler/commit/688bbfb33cee10f15647a250a52b5a2dafb8c7ca

The way to specify the source code file is to use `http://localhost:8080/?sample` or `http://localhost:8080/?cleanEHR`